models/ngspice/testing/regression/.spiceinit

```diff
@@ -1,2 +1,3 @@
 ************
 set ngbehavior=hs
+set numthreads=8
```

Comment:
Changing the number of threads is unrelated to the resistor change?

Reply:
It helps speed up the testing.

Reply:
I'm not claiming it doesn't speed up testing.

I'm saying a change which speeds up testing should not be mixed into a change which is suppose to fix resistors.

Please split the changes into one pull request per feature change